### PR TITLE
feat(tui): add Vim motions support for task list selection

### PR DESCRIPTION
**Pull request by AlvaroParker**
**Summary:**

This PR introduces support for basic Vim-style key bindings within the `tui` component of TurboRepo, specifically for pane navigation. The goal is to improve navigation efficiency for users familiar with Vim-like key motions.

**Key Changes:**
- Added support for `h`, `j`, `k`, and `l` Vim keybinds for interactive mode:
  - **`Ctrl+h`**: Exit interactive mode.
  - **`Ctrl+j`**: Move down.
  - **`Ctrl+k`**: Move up.
  - **`Ctrl+l`**: Enter interactive mode.
- General Vim-style motions:
  - **`j`**: Move down.
  - **`k`**: Move up.
  - **`l`**: Enter interactive mode.

**Motivation:**
Adding these motions provides a more intuitive interface for users familiar with Vim commands, enhancing navigation within the pane section of the TurboRepo UI.

*[Related discussion](https://github.com/vercel/turborepo/discussions/9227)*

### CODE REVIEW

**Comment by AlvaroParker** [comment]
Don't really know why the workflows are failing. I updated my branch, waiting for approval : )

**Comment by dimitropoulos** [comment]
hey just to keep you in the loop, I was reviewing this just now and we're gonna talk about it at our next team meeting and get back to you: there are some future plans for some of these keys (beyond just `j` and `k`) and we need to make sure we don't disrupt those plans.

**Comment by anthonyshew** [comment]
Hey, @AlvaroParker! Just wanted to see if you're interested in finishing this one up. If not, we can remove the unwanted functionality on your behalf.

Thanks, excited for this!

**Comment by AlvaroParker** [comment]
Hey @anthonyshew, sorry for not making the changes earlier—it’s been a busy couple of weeks. I’ve committed the changes to just use `j` and `k`. Let me know if I missed anything!

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: f7b8a0bc78ca126018ab5536fb7beb45cc6a53d4

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by anthonyshew — APPROVED** [review]
No worries! Thanks for taking the time. Appreciate it!

**Pull request merged**
